### ISSUE TITLE
Mobile menu toggle doesn't work

### ISSUE THREAD

**Issue by narrenfrei** [google/docsy#521]
I've noticed, that the "hamburger button" to toggle the sidebar menu on mobile devices doesn't work any more. Maybe a side effect of updating bootstrap?

**Comment by narrenfrei** [comment]
I've now copied the official example for "collapse buttons" from the bootstrap site (https://getbootstrap.com/docs/4.0/components/collapse/) into a docsy partial to test the buttons. The example also doesn't work correctly. So I think, it has really something to do with the bootstrap update (#516).

```html
<p>
  <a class="btn btn-primary" data-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
    Link with href
  </a>
  <button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample">
    Button with data-target
  </button>
</p>
<div class="collapse" id="collapseExample">
  <div class="card card-body">
    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident.
  </div>
</div>
```

Anybody (maybe @deining ?) here who has more experience with bootstrap, because I have no.

**Comment by deining** [comment]
Mea culpa! The SRI hash provided in #516 was incorrect. This was fixed in #522.
Sorry for any inconvenience caused!

**Comment by narrenfrei** [comment]
No problem and thanks for your fast reaction.

**Comment by LisaFC** [comment]
Fixed in #522
